### DISCUSSION TITLE
Recommended videos irrelevant to the topic and from the wrong region

### DISCUSSION THREAD

**Discussion by Felix2-2** [FreeTubeApp/FreeTube#4433] [Q&A] [answered]
Hello, 

The recommended videos side bar on the right often shows videos completely unrelated to the topic of the video I'm currently watching. Also, it sometimes shows videos from the wrong region and in the wrong language.  

For example, I am watching a video about history and in the recommended I get political news from Poland or tutorials for kids. 

My settings: 
![image](https://github.com/FreeTubeApp/FreeTube/assets/153457109/08d4559a-4792-4fc2-85b8-e52f6902565b)

For reference, I do live in Poland currently but in my Windows region and language settings everything is also set to US.

**Comment by absidue** [comment, marked as answer]
The recommended videos come from YouTube, so if YouTube decides to recommend stuff based on your physical location and you don't like that, all you can do is change your IP address with a VPN.
